migrations/migrations/20180917134433-create-users.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('usernames', {
```

Comment:
change to users here

Reply:
ce1e674d2b13fc863c46dfcbc76348e5325c23b7